tests/Home.js

```diff
@@ -26,108 +24,131 @@ describe("Home page", () => {
     await (await HamburgerMenu.aboutButton).click();
     await expect(await Menu.title).toHaveText("About Us");
 
-    await browser.execute(() => window.scrollTo(0, 500));
+    await HomePage.scrollDown();
+    await (await HamburgerMenu.hamburgerMenuButton).waitForDisplayed();
     await (await HamburgerMenu.hamburgerMenuButton).click();
+    await (await HamburgerMenu.privacyPolicyButton).waitForDisplayed();
     await (await HamburgerMenu.privacyPolicyButton).click();
     await expect(await Menu.title).toHaveText("Privacy Policy");
 
-    await browser.execute(() => window.scrollTo(0, 500));
+    await HomePage.scrollDown();
+    await (await HamburgerMenu.hamburgerMenuButton).waitForDisplayed();
     await (await HamburgerMenu.hamburgerMenuButton).click();
+    await (await HamburgerMenu.dmcaButton).waitForDisplayed();
     await (await HamburgerMenu.dmcaButton).click();
     await expect(await Menu.title).toHaveText("DMCA Policy");
 
-    await browser.execute(() => window.scrollTo(0, 500));
+    await HomePage.scrollDown();
+    await (await HamburgerMenu.hamburgerMenuButton).waitForDisplayed();
     await (await HamburgerMenu.hamburgerMenuButton).click();
+    await (await HamburgerMenu.advertisingButton).waitForDisplayed();
     await (await HamburgerMenu.advertisingButton).click();
     await expect(await Menu.title).toHaveText("Advertising Interest");
 
-    await browser.execute(() => window.scrollTo(0, 500));
+    await HomePage.scrollDown();
+    await (await HamburgerMenu.hamburgerMenuButton).waitForDisplayed();
     await (await HamburgerMenu.hamburgerMenuButton).click();
+    await (await HamburgerMenu.closeMenu).waitForDisplayed();
     await (await HamburgerMenu.closeMenu).click();
   });
 
-  it("Footer", async () => {
-    await (await HomePage.aboutFooterButton).waitForDisplayed();
+  it("Check the first part of footer functionality", async () => {
+    await HomePage.scrollDown();
+    await (await HomePage.aboutFooterButton).waitForClickable();
     await (await HomePage.aboutFooterButton).click();
+    await (await Menu.title).waitForDisplayed();
     await expect(await Menu.title).toHaveText("About Us");
     await HomePage.backPage();
 
-    await HomePage.privacyPolicyFooterButton.click();
+    await HomePage.scrollDown();
+    await (await HomePage.privacyPolicyFooterButton).waitForClickable();
+    await (await HomePage.privacyPolicyFooterButton).click();
+    await (await Menu.title).waitForDisplayed();
     await expect(await Menu.title).toHaveText("Privacy Policy");
     await HomePage.backPage();
+  });
 
+  it("Check the last part of footer functionality", async () => {
+    await HomePage.scrollDown();
+    await (await HomePage.dmcaFooterButton).waitForClickable();
     await (await HomePage.dmcaFooterButton).click();
+    await (await Menu.title).waitForDisplayed();
     await expect(await Menu.title).toHaveText("DMCA Policy");
     await HomePage.backPage();
 
+    await HomePage.scrollDown();
```

Comment:
I don't see this function added on this PR? it is on master already?

Reply:
If you  merge this PR and this function is not on the main branch you need to add the file where you wrote this function: scrollDown() on this PR

Reply:
I added the function in this PR